src/odemis/acq/move.py

```diff
@@ -54,23 +54,31 @@ def getCurrentPositionLabel(current_pos, stage):
     stage_active = stage_md[model.MD_FAV_POS_ACTIVE]
     stage_active_range = stage_md[model.MD_POS_ACTIVE_RANGE]
     stage_coating = stage_md[model.MD_FAV_POS_COATING]
+    stage_alignment = stage_md[model.MD_FAV_POS_ALIGN]
+    stage_sem_imaging = stage_md[model.MD_FAV_POS_SEM_IMAGING]
```

Comment:
Maybe check if these metadata exist before using them. I think there is already check for the active, deactive and coating positions in the cryo chamber tab class. You could check for the new two

Reply:
In `CryoSwitchSamplePosition` there's check that all axes exist for these metadata.